doc/docs/master/deploy-manage/deploy/deploy-pachyderm-jupyterhub.md

```diff
@@ -30,124 +39,69 @@ deployed on the same Kubernetes cluster. You deploy Pachyderm by
 using the pachctl `deploy command` and JupyterHub by running
 our deployment script as described below. After deployment, you log in
 to JupyterHub with your Pachyderm user and interact with Pachyderm
-from within the Jupyter UI by using Pachyderm Python client.
+from within the Jupyter UI by using the Pachyderm Python client.
 
 ## Prerequisites
 
-Before you can deploy JupyterHub with `python-pachyderm`, you need to
-install Pachyderm on a supported Kubernetes platform and configure
-other prerequisites listed below.
-
-### Deploy Pachyderm
-
-You need to deploy Pachyderm as described in
-[Deploy Pachyderm](../../deploy/)
-in of the supported platforms:
-
-- Google Kubernetes Engine (GKE) with Kubernetes v1.13
-- Amazon Elastic Container Service (EKS) with Kubernetes v1.13
-- Docker Desktop for Mac with Kubernetes v1.14
-
-!!! note
-    Kubernetes v1.16 is not supported, including Minikube deployments.
-    If you already have a local Pachyderm deployed with Kubernetes v1.16,
-    you might not be able to deploy JupyterHub on the same Kubernetes
-    cluster by using the Jupyter deployment script.
-
-For more information about JupyterHub requirements for Kubernetes,
-see [Zero to JupyterHub with Kubernetes](https://zero-to-jupyterhub.readthedocs.io/en/latest/).
-
-### Enable Pachyderm Access Controls (Optional)
-
-You can enable Pachyderm access controls to use Pachyderm credentials
-to log in to JupyterHub. However, this
-is not required, and if you prefer to use JupyterHub authentication
-instead, skip this section.
+Before deploying Pachyderm IDE, configure the following prerequisites:
 
-Pachyderm Access Controls is an enterprise feature. You need to have
-an active Enterprise token to enable it.
+* Deploy Pachyderm 1.11.0 or later as described in
+[Deploy Pachyderm](https://docs.pachyderm.com/latest/deploy-manage/deploy/)
+on a supported Kubernetes platforms:
 
-To enable Pachyderm authentication, follow the steps in
-[Configure Access Controls](https://docs.pachyderm.com/latest/enterprise/auth/auth/)
+  - Google Kubernetes Engine (GKE) with Kubernetes v1.13
+  - Amazon Elastic Container Service (EKS) with Kubernetes v1.13
+  - Docker Desktop for Mac with Kubernetes v1.14
```

Comment:
The Pachyderm IDE includes a patch that makes it compatible with newer versions of kubernetes, so this is no longer relevant

Reply:
corrected